docs/health-api-specs/contracts/referral-management.yml

```
@@ -1,810 +1,1879 @@
-swagger: '2.0'
+openapi: 3.0.1
 info:
-  version: 1.0.0
-  title: Referral Management System
-  contact:
-    name: egovernments foundation
-    email: info@egovernments.org
-schemes:
-  - https
-x-common-path: https://raw.githubusercontent.com/egovernments/egov-services/master/docs/common/contracts/v1-0-0.yml
+  title:  Referral Management System
```

YAMLlint (1.35.1)

[error] too many spaces after colon

(colons)